### ISSUE TITLE
Feature request: turning individual segments on and off

### ISSUE THREAD

**Issue by ruslanbd** [RobTillaart/TM1637_RT#27]
Blinking with individual segments is a much-needed feature. 
For example, when adjusting a clock, blinking with minute digits when setting up minutes, and blinking with hour digits when setting up hours.
It would be nice to have such a function to keep the code clean and readable, instead of fiddling with displayRaw(), space characters and buffers.

**Comment by RobTillaart** [comment]
Thanks for this feature request. 
Need to find some hardware to test and think how to implement this in a nice generic way. 
Might take a few days

**Comment by RobTillaart** [comment]
@ruslanbd 
Do you need to hide one segment or be able to hide multiple segments too?

**Comment by ruslanbd** [comment]
@RobTillaart 
Multiple would be nice, for example when blinking a digit or two when highlighting a part of the display's output.
After all, the more flexible the functionality is, the better.

**Comment by RobTillaart** [comment]
@ruslanbd 
Created a develop branch with some experimental extensions to solve this issue.

- add **hideSegment(n)**  n = 0..8
- add **displayRefresh()** to refresh last written data.

To hide a segment you call e.g. TM.hideSegment(2);
and to let it reappear again you either call **displayRefresh()**  or **displayTime()** or another display.

to let it blink you need to call hide and refresh alternatingly as the library does (and will) not support background threads.

**Comment by RobTillaart** [comment]
wrote a multi segment hide function, please verify if it works.

a 1 bit indicates a blank segment , so a mask of 0b00001001 should blank segment 0 and 3.
**Q: is a 0 for blank more intuitive?  1 == ON, 0 == OFF?**

Note:  "refresh all" ==> set mask to 0x00

```cpp
void TM1637::hideMultiSegment(uint8_t mask)
{
  //  debug info
  Serial.println(__FUNCTION__);
  Serial.println(mask, HEX);

  uint8_t tmp[8];
  for (int i = 0; i < 8; i++) 
  {
    if ((mask & 0x01) == 0x01) tmp[i] = TM1637_SPACE;
    else tmp[i] = _data[i];
    mask >= 1;
  }
  displayRaw(tmp, _lastPointPos);
}
```

**Comment by RobTillaart** [comment]
uploaded above **hideMultiSegment(mask)** into the develop branch

**Comment by ruslanbd** [comment]
Looks good! I'll verify in the meantime and let you know

**Comment by ruslanbd** [comment]
I think 1 for a hidden segment is more intuitive, since the function is hiding the segmemts.

**Comment by RobTillaart** [comment]
@ruslanbd 

cleaned up the code a bit and created a PR for release 0.3.8 
to be merged later

**Comment by RobTillaart** [comment]
Solved and released in version 0.3.8
